src/Aspire.Hosting.Testing/DistributedApplicationTestingBuilder.cs

```diff
@@ -297,7 +294,12 @@ private static DistributedApplicationBuilder CreateInnerBuilder(
                 builder.Services.ConfigureHttpClientDefaults(http => http.AddStandardResilienceHandler());
             }
 
-            return builder;
+            if (!builder.Configuration.GetValue("Aspire:Store:Path", false))
+            {
+                builder.Configuration["Aspire:Store:Path"] = _tempAspireStorePath = Path.GetTempPath();
```

Comment:
I think we should just do this in our tests, not for customer tests. Customer tests should use the AppHost's `obj` directory, just like a normal App IMO.